picamraw/main.py

```diff
@@ -244,7 +244,13 @@ def _unpack_10bit_values(pixel_bytes_2d):
 
     # In each row, split up every 5th byte and unpack it into the low 2-bits of the four preceding bytes
     for byte in range(4):
-        data[:, byte::5] |= ((data[:, 4::5] >> ((4 - byte) * 2)) & 3)
+        fifth_bytes = pixel_bytes_2d[:, 4::5]
```

Comment:
I think it would be a bit clearer if you pulled this out of the for loop (i.e. defined it alongside `data` on line 244 above.)

Reply:
done